Sources/Danger/DangerUtils.swift

```diff
@@ -104,6 +106,26 @@ public struct DangerUtils {
             }
         }
     }
+
+    /// Converts an asynchronous function to synchronous.
+    ///
+    /// - Parameter body: The async function must be called inside this body and closure provided as parameter must be executed on completion
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/DangerUtils.swift;
  Line: 112;
-->

- :warning: Line should be 120 characters or less: currently 141 characters (`line_length`)